doc/guide/start.rst

```diff
@@ -0,0 +1,453 @@
+to start a Flux instance.  The ``--pty`` option gives Flux's interactive
+initial program a proper terminal.
+
+.. code-block:: console
+
+  $ salloc -N2
+  salloc: Pending job allocation 1505790
+  salloc: job 1505790 queued and waiting for resources
+  salloc: job 1505790 has been allocated resources
+  salloc: Granted job allocation 1505790
+  salloc: Waiting for resource configuration
+  salloc: Nodes quartz[2257-2258] are ready for job
+  $ srun -N2 --pty flux start
+  $ flux uptime
+   10:04:05 run 6.6s,  owner alice,  depth 0,  size 2
+  $ flux resource info
+  2 Nodes, 72 Cores, 0 GPUs
+  $ flux run -N2 hostname
+  quartz2257
+  quartz2258
+  $ exit
+  $ exit
+  salloc: Relinquishing job allocation 1505790
+  salloc: Job allocation 1505790 has been revoked.
+  $
+
+After typing the same two Flux commands that were demonstrated in
+:ref:`test_instance`, we exit the Flux interactive shell, then the
+Slurm one, which gives up the Slurm allocation.
+
+Conceptually, what's going on here is Slurm has granted a resource
+allocation to Flux, which then parcels it out to the jobs submitted to it.
+
+The above can be accomplished a little more succinctly by just running
+srun directly.  Then the session looks like this:
+
+.. code-block:: console
+
+  $ srun -N2 --pty flux start
+  srun: job 1505791 queued and waiting for resources
+  srun: job 1505791 has been allocated resources
+  $ flux uptime
+   10:05:51 run 4.7s,  owner alice,  depth 0,  size 2
+  $ flux resource info
+  2 Nodes, 72 Cores, 0 GPUs
+  $ flux run --label-io -N2 hostname
+  0: quartz2257
+  1: quartz2258
+  $ exit
+  $
+
+To run non-interactively, we can drop the ``--pty`` option and use our
+workload script for the initial program:
+
+.. code-block:: console
+
+  $ srun -N2 flux start ./workload.sh
+  srun: job 1505795 queued and waiting for resources
+  srun: job 1505795 has been allocated resources
+   10:07:12 run 4.6s,  owner alice,  depth 0,  size 2
+  2 Nodes, 72 Cores, 0 GPUs
+  0: quartz25
+  1: quartz26
+
+Finally, Slurm batch execution of a Flux workload can be accomplished
+by wrapping the srun command in a batch script we will name ``batch.sh``:
+
+.. code-block:: shell
+
+  #!/bin/sh
+  #SBATCH -N2
+  srun flux start ./workload.sh
+
+Whee!
+
+.. code-block:: console
+
+  $ sbatch ./batch.sh
+  Submitted batch job 1505846
+  $ cat slurm-1505846.out
+   10:09:57 run 4.8s,  owner alice,  depth 0,  size 2
+  2 Nodes, 72 Cores, 0 GPUs
+  0: quartz47
+  1: quartz48
+  $
+
+Inserting Flux between Slurm and a real workload script or workflow executor
+could have some advantages, such as:
+
+- Workload scripts that use Flux commands or the Flux python API can be made
+  portable to other systems that don't use Slurm as their primary resource
+  manager, as long as they can launch a Flux instance.
+
+- High throughput workloads don't have to bother the central Slurm controller
+  with every step, so they are less of a burden on the system and may run
+  faster.
```

Comment:
maybe need to explain, "All slurm steps from all jobs in the system go through the central Slurm controller ...".